src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -133,7 +153,48 @@ const LightUpChinatownPage = () => {
     </React.Fragment>
   );
 };
+// left: 42%;
+// top: 70%;
+const Button = styled.div`
+  line-height: 40px;
+  width: 15%;
+  min-width: 150px;
+  max-width: 220px;
+
+  position: absolute;
+  top: 67%;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  cursor: pointer;
+
+  font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  letter-spacing: 0.12em;
+
+  box-shadow: 0 0 0.5pt 0.5pt black;
+  background-color: #ffffff;
+  border-radius: 100px;
+  padding: 10px 10px 10px 10px;
+  @media (${phoneScreens}) {
+    font-size: 14px;
```

Comment:
How come the font changes on different phone sizes?

Reply:
The size 18 looked big on my iphone but i'll put both size 14 and 18. Go back to size 18?
![image](https://user-images.githubusercontent.com/20375084/99290857-e4e71480-280c-11eb-9895-99826295b2df.png)
![image](https://user-images.githubusercontent.com/20375084/99290877-ee707c80-280c-11eb-962b-6205a4d058dd.png)